openreview/arr/webfield/homepageWebfield.js

```diff
@@ -0,0 +1,45 @@
+// Webfield component
+const tabs = [{
+    name: 'Your Consoles',
+    type: 'consoles'
+  }]
+  
+  tabs.push({
+    name: 'Anonymous Pre-prints',
+    query: {
+      'content.venueid': domain.content.submission_venue_id?.value
+    },
+    options: {
+      postQuery: {
+        'readers': 'everyone'
+      },
+      hideWhenEmpty: true,
+			pageSize: 1000
```

Comment:
Shouldn't this indentation match `hideWhenEmpty` (does it need to be unindented)?